champagne/pi/float.go

```diff
@@ -182,28 +208,86 @@ func saltDeployment() error {
 	// wait for deployment stuff to finish
 	log.Printf("waiting for deployment to finish (5 minutes).\ncur time: %v", time.Now())
 
+	if err := waitForFile(context.TODO(), DockerComposeFile, true); err != nil {
+		return fmt.Errorf("docker compose file never showed up: %s", err)
+	}
+
+	dockerCompose, err := ioutil.ReadFile(DockerComposeFile)
+	if err != nil {
+		return fmt.Errorf("unable to read docker compose file: %s", err)
+	}
+
+	var dockers dcompose
+
+	if err = yaml.Unmarshal(dockerCompose, &dockers); err != nil {
+		return fmt.Errorf("unable to unmarshal docker-compose.yml: %s", err)
+	}
+
+	dockerNum := 0
+
+	//this is the number of dockers that should be running
+	for range dockers.Services {
+		dockerNum++
+	}
+
+	cli, err := client.NewEnvClient()
+	// cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		return fmt.Errorf("unable to get docker client: %s", err)
+	}
+
+	runningDockers, err := cli.ContainerList(context.TODO(), types.ContainerListOptions{})
+	if err != nil {
+		return fmt.Errorf("uanble to get list of running docker containers: %s", err)
+	}
+
```

Comment:
i think this needs to get moved into the for loop below? so that the number of running containers will update